aicsimageio/readers/lif_reader.py

```diff
@@ -75,6 +76,9 @@ def __init__(
         image: types.PathLike,
         chunk_dims: Union[str, List[str]] = DEFAULT_CHUNK_DIMS,
         fs_kwargs: Dict[str, Any] = {},
+        is_x_flipped: bool = False,
```

Comment:
Since users pass a reference to just the type of reader not an instantiated reader is this effectively useless? I appreciate any advice on what to do with this, at the very least I think keeping it around somewhere to expose that by default we are doing something is a good idea.

Reply:
I sort of remember we had a way of forwarding AICSImage constructor kwargs down the reader instantiation.

Reply:
Also, I just did a quick check of the bioformats code and found this https://github.com/ome/bioformats/blob/develop/components/formats-gpl/src/loci/formats/in/LIFReader.java#L2053

Not sure if it's worth it but `readlif` has this `info` parameter (https://github.com/nimne/readlif/blob/master/readlif/reader.py#L74) and maybe that gives access to the confocal settings to see whether theres a flipX?

Reply:
> I sort of remember we had a way of forwarding AICSImage constructor kwargs down to the reader instantiation.

This seemed to work! 
```
img = AICSImage("~/Downloads/20210428_24w_L929_Ho_B2C3.lif", reader=LifReader, is_x_flipped=True, is_y_flipped=True, is_x_and_y_swapped=False)
```

> Not sure if it's worth it but readlif has this info parameter

This seems to just be a duplicate of the existing metadata it offers (I doubled checked). I gave up looking for the data within `readlif` after I found [this claim that the package does not support it](https://github.com/nimne/readlif/blob/master/README.md?plain=1#L31). Does this seem worth opening an issue with them?

Reply:
Wow!  Since they asked for it, I'd say yes we should open an issue.

Reply:
[Done](https://github.com/nimne/readlif/issues/45) ✅